docs/guide/wallets.md

```diff
@@ -76,6 +76,14 @@ For the invoice to work you must have a publicly accessible URL in your LNbits.
 - `OPENNODE_API_ENDPOINT`: https://api.opennode.com/
 - `OPENNODE_KEY`: opennodeAdminApiKey
 
+### Blink
+
+For the invoice to work you must have a publicly accessible URL in your LNbits. No manual webhook setting is necessary. You can generate an Blink.sv access token here: https://www.blink.sv/en/api
```

Comment:
Suggestion to change to the correct link of the Blink Dashboard and point to to the technical documentation.

```suggestion
For the invoice to work you must have a publicly accessible URL in your LNbits. No manual webhook setting is necessary. You can generate a Blink API key after logging in or creating a new Blink account at: https://dashboard.blink.sv. For more info visit: https://dev.blink.sv/api/auth#create-an-api-key```